config/locales/views/groupings/en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  views:
+    groupings:
+      manage: "Manage"
+      set_team_settings: "Set of teams settings"
+      drag_and_drop_student_avatars: "You can <b>drag and drop</b> the student avatars to move them from one team to another."
```

Comment:
Hmmm, it feels wrong to pull style information into the i18n locales. I'm not sure there's an alternative here besides splitting the string into 3. @johndbritton do you have experience with this?